app/views/timecards/_timecard.html.erb

```diff
@@ -1,8 +1,11 @@
 <tr>
-  <td><%= link_to timecard.status, timecard_path(timecard) %></td>
+  <td><%= link_to timecard.status, timecard_path(timecard), id: "js-timecard-status-id-#{timecard.id}" %></td>
   <td><%= timecard.start_time.strftime('%a %b %d %Y %k:%M') if timecard.start_time %></td>
   <td><%= timecard.end_time.strftime('%a %b %d %Y %k:%M') if timecard.end_time%></td>
   <td><%= timecard.duration %></td>
   <td><%= truncate(timecard.description, length: 24) %></td>
-  <td><%= table_button_link 'Edit', edit_timecard_path(timecard) if can? :edit, timecard %></td>
+  <td>
+    <%= button_to 'Verify', timecard_verify_path(timecard), form_class: 'timecard-verify', id: "js-timecard-verify-btn-id-#{timecard.id}", class: 'js-timecard-verify-btn btn btn-xs btn-success' if (can? :edit, timecard) && timecard.status != 'Verified' %>
```

Comment:
Would need to change the conditional here. 
`if (can? :verify, timecard) && timecard.status == 'Unverified'